javascript/javascript1/week4/homework.js

```diff
@@ -0,0 +1,135 @@
+let introducedName = "";
+let toDos = [];
+function getReply(command) {
+  if (command.startsWith("Hello my name is")) {
```

Comment:
Remember that this test and the following ones are case-sensitive. They would not work if a user types `hello my name is`. How could you handle such a situation?